RecoTracker/LSTCore/standalone/Makefile

```diff
@@ -14,7 +14,7 @@ INCLUDEFLAGS= -ILST -I$(shell pwd) -Icode -Icode/core -I${ALPAKA_ROOT}/include -
 ifdef CMSSW_RELEASE_BASE
 INCLUDEFLAGS:= ${INCLUDEFLAGS} -I${CMSSW_RELEASE_BASE}/src
 endif
-LDFLAGS     = -g -O2 $(LSTLIB) -L${TRACKLOOPERDIR}/LST $(shell rooutil-config --libs) $(shell root-config --libs) -L${CMSSW_BASE}/lib/${SCRAM_ARCH} -lFWCoreMessageLogger
+LDFLAGS     = -g -O2 $(LSTLIB) -L${TRACKLOOPERDIR}/LST $(shell rooutil-config --libs) $(shell root-config --libs) -L${CMSSW_BASE}/lib/${SCRAM_ARCH} -L${CMSSW_RELEASE_BASE}/lib/${SCRAM_ARCH} -lFWCoreMessageLogger
```

Comment:
a few lines above we have an `ifdef CMSSW_RELEASE_BASE`, that I added at some point.
This may be a better logic to extend, if this env is defined.
IIUC, we still have two paths of calling setup.sh from the local checkout vs from an existing central location.

Reply:
Thanks @slava77. That sounds very reasonable. I'll try rearranging